ui/narrative/methods/annotate_contigs/spec.json

```diff
@@ -23,11 +23,29 @@
         },
         {
             "id": "scientific_name",
-            "optional": true,
+            "optional": false,
```

Comment:
Do you always have to know the taxonomy of what you're trying to annotate? Can you annotate something where you don't know the taxonomy?

Reply:
I think you can annotate it without a taxonomy. It also helps avoid situations where a valid taxonomic classification exists, but is not in system

Reply:
So if you can annotate without taxonomy, we want to keep this optional?

Reply:
Yes. Its only used for the scientific name field of the output genome. It defaults to "Unknown"

Reply:
So are you going to revert this back to `"optional": true`?

Reply:
i might be colorblind